ckanext/geocat/metadata.py

```diff
@@ -400,7 +413,10 @@ def _create_dist_copy(self, orig_dist, access_url):
 
     def _get_dataset_metadata(self, xml):
         dataset = GeocatDcatDatasetMetadata()
-        dataset_meta = dataset.load(xml)
+        dataset_meta, raw_meta = dataset.load(xml, include_raw=True)
+
+        # copy rights from raw metadata
```

Comment:
I would prefer if the comment explained it a bit more clear why we have to get the rights-data from the dataset. Something like:
As Geocat has the rights information on dataset-level, this workaround is needed so we can save it to distributions....